model_analyzer/result/run_config_measurement.py

```diff
@@ -380,6 +393,41 @@ def __lt__(self, other):
 
         return not self.is_better_than(other)
 
+    def is_passing_constraints(self):
+        """
+        Returns true if all model measurements pass
+        their respective constraints
+        """
+        return ConstraintManager.satisfies_constraints(
+            self._model_config_constraints, self)
+
+    def compare_measurements(self, other):
+        """
+        Compares two RunConfig measurements based on each
+        ModelConfigs weighted metric objectives and the
+        ModelConfigs weighted value within the RunConfig
```

Comment:
do we have a RunConfig anywhere here?  Let's not couple the functionality with the expected usage

Reply:
This should be RCM. Updated the comment